spec/models/list_spec.rb

```diff
@@ -42,4 +42,12 @@ def create_list_with_books(*books)
       end
     end
   end
+
+  it "sets a slug when saved" do
+    record = create(:list, name: "Foo Bar")
+
+    result = record.slug
+
+    expect(result).to eq("foo-bar")
```

Comment:
I considered using shared examples here but it's `name` in some places and `title` in the other, so I'd have to pass that as a parameter which reduced the readability.